index.html

```diff
@@ -165,6 +165,21 @@ <h2>Skills</h2>
         </li>
       </ul>
       </section>
+      <section class="form">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form name="form1" action="https://formspree.io/f/myyovzdk" method="post">
+          <label >
+            <input type="text" name="name" id="name"  placeholder="Yeremias" maxlength="30" required>
+          </label> <br>
+          <label >
+           <input type="email" name="email-address" id="email" placeholder="natajayanj14@gmail.com" maxlength="30" required>
```

Comment:
- [ ] [optional] the inputs name and email are not the same as the ones in the Figma design please try to stick to the Figma design 
![image](https://user-images.githubusercontent.com/61790587/172441252-38b0dd59-d7db-42c9-a5ea-27fedaf3dd58.png)